changes/4811.changed

```diff
@@ -1 +1 @@
-Changed the behavior of tree model tables: now they are sortable, and after sorting is applied, all hierarchy indentations are removed.
\ No newline at end of file
+Changed the behavior of tree model tables and Prefix table: now they are sortable, and after sorting is applied, all hierarchy indentations are removed.
```

Comment:
This needs to be moved to a separate change fragment as the original fragment has already been incorporated into 2.0.0b1 release notes.